tests/make_observations.py

```diff
@@ -137,10 +139,12 @@ def make_observations(
     ephemeris_dfs = []
     for i, orbit in enumerate(orbits):
         initial_epoch = Time(orbit._epoch, scale="tt", format="mjd")
-        # Observation times are defined at the center of the exposure (for now)
-        observation_times = (
-            initial_epoch.utc.mjd + dts + exposure_duration / 86400 / 2.0
-        )
+
+        # Calculate a random offset from the start of the exposure
+        # to give each observation a unique obervation time
+        offset_from_start = np.random.uniform(dts, dts + exposure_duration / 86400)
+        observation_times = initial_epoch.utc.mjd + offset_from_start
+        exposure_ids = [f"{obs_i}_{k:06d}" for k, obs_i in enumerate(observatory_codes)]
```

Comment:
Why is this inside the loop over `orbits`? It seems like it should be done once for all orbits during the initial setup.

Reply:
No particularly good reason other than it seemed better to add different offsets for different orbits. In other words, I wanted each orbit to have unique dts to complement the fact that they already are not constrained to have the same initial epochs (by design). In the event, however, that they do have the same initial epoch then I wanted one additional layer to separate their observation times.

Reply:
That's not how real data would work, though, right? Wouldn't the observations always have consistent timestamps, even for orbits with the same or different epochs?

Reply:
It's unlikely two moving objects observed in the same exposure will have the same time unless they were exactly positioned along the axis perpendicular to the motion of the shutter so instead this synthetic data doesn't model the shutter and just makes it random for each orbit within the confines of an exposure. 

I think what it comes down to is that the synthetic data isn't modeling the footprint of images on the sky. The sample orbits are all over the place and so two orbits observed on the sky may be at polar opposite locations but still be within the same "exposure". We are really just testing that given an orbit and some dummy observation times do we extract all of the observations with the input orbit? The times, the cadences, observatory codes, exposure durations, etc.. are all potential variables on how this test may perform but they may not represent reality for any given set of observations from a real telescope.